tests/test_files.py

```diff
@@ -17,7 +17,6 @@ class G(GoodConf):
         g = G()
         g.load()
     mocked_load_config.assert_called_once_with(str(path))
-    assert g._config_file == str(path)
```

Comment:
Removed and similar below because `_config_file` no longer exists and the assertion doesn't provide much value anyways since we check that `load_config` is called with that path.